jinja/plugins.html

```diff
@@ -31,22 +31,33 @@ <h4 class="card-header juicy-peach-gradient">default.qubit</h4>
                 </a>
             </div>
             <div class="card plugin-card">
-                <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.plugins.default_gaussian.DefaultGaussian.html">
-                    <h4 class="card-header juicy-peach-gradient">default.gaussian</h4>
+                <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.plugins.default_qubit_tf.DefaultQubitTF.html">
+                    <h4 class="card-header juicy-peach-gradient">default.qubit.tf</h4>
                     <div class="card-body">
                         <p class="card-text">
-                            A simple quantum photonic simulator written in Python.
-                            <br><br>A good choice for optimizating photonic systems.
+                            A state-vector qubit simulator written using TensorFlow and supporting classical backpropagation. <br><br>A good choice when the number of parameters to be optimized is large.
                         </p>
                     </div>
                 </a>
             </div>
             <div class="card plugin-card">
-                <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.plugins.default_qubit_tf.DefaultQubitTF.html">
-                    <h4 class="card-header juicy-peach-gradient">default.qubit.tf</h4>
+                <a href="https://pennylane-lightning.readthedocs.io">
+                    <h4 class="card-header juicy-peach-gradient">lightning.qubit (beta)</h4>
                     <div class="card-body">
                         <p class="card-text">
-                            A state-vector qubit simulator written using TensorFlow and supporting classical backpropagation. <br><br>A good choice when the number of parameters to be optimized is large.
+                            A state-vector qubit simulator written with a C++ backend.
+                            <br><br>A good choice if you want to run faster than default.qubit.

```

Comment:
Minor comment: This particular 'good choice' feels somewhat more subjective than the others. Perhaps we can extend the `default.qubit` 'good choice' phrasing:

```suggestion
                            <br><br>A good choice for optimizations with a large number of qubits and a moderate number of parameters.
```

(in the case of a large number of parameters, a backprop device will be a better choice)

Reply:
```suggestion
                            <br><br>A fast implementation of default.qubit.
```
@josh146, do you think it needs a "A good choice" at the start? For me, `lightning.qubit` shines in the same regime as `default.qubit`, e.g. moderate numbers of qubits/params. Indeed, we see for going up to 8 qubits that we're having a bit of an overhead, so I'd be hesitant to say it's a good choice for large numbers of qubits.

Reply:
Ah, good point - I think I'm mostly trying to avoid a case where most users see 'fast', and end up using `lightning.qubit` for everything.

Reply:
True :thinking: how about the new suggestion, which isn't comparative (i.e. "faster"), it just says it's fast :grin:

Reply:
Because somehow we need to say that `lightning.qubit` is fast as it's kind of the selling point, also maybe the "beta" warning is enough to make people a bit more cautious